### ISSUE TITLE
pull request #294 break latexpdf with 'howto' class

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1607]
https://bitbucket.org/birkenfeld/sphinx/pull-request/294 "Wrap float environment on writing literal_block to latex" break latexpdf processing if the sphinx project uses `howto` documentclass.

```
$ make latexpdf
sphinx-build -b latex -d _build/doctrees   . _build/latex
Running Sphinx v1.3b1

-snip-

Package hyperref Message: Driver (autodetected): hpdftex.

(/usr/local/texlive/2014/texmf-dist/tex/latex/hyperref/hpdftex.def
(/usr/local/texlive/2014/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty))
(/usr/local/texlive/2014/texmf-dist/tex/latex/oberdiek/hypcap.sty)
(/usr/local/texlive/2014/texmf-dist/tex/latex/float/float.sty)

! LaTeX Error: No counter 'chapter' defined.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.529 \newfloat{literal-block}{htbp}{loc}[chapter]
```

This error hadn't been raised before PR#294 was merged.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1607
- Originally reported by: Takayuki Shimizukawa
- Originally created at: 2014-10-27T07:53:03.131

**Comment by shimizukawa** [comment]
_From Valentine Reboul on 2014-11-21 17:18:46+00:00_

Same issue here. Can't use 'howto' anymore. any news about it ?

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-11-25 01:38:18+00:00_

https://bitbucket.org/birkenfeld/sphinx/pull-request/317 will be merged soon.

**Comment by shimizukawa** [comment]
_From Valentine Reboul on 2014-11-25 13:44:26+00:00_

Great news !! Thx ! :)

**Comment by shimizukawa** [comment]
_From Takeshi Komiya on 2014-11-27 07:35:13+00:00_

Fix #1607: Sphinx crashes on building latexpdf with "howto" class

→ <<cset 0f1d8433e7b95e2b9bb0097ae1ca305575ed0411>>
